Disable optimize separately (#291)
camunda/camunda-platform-helm: charts/camunda-platform/README.md

| Section | Parameter | Description | Default |
|-|-|-|-|
| `identity`| |  Configuration for the identity sub chart. | |

> | | `enabled` |  If true, the identity deployment and its related resources are deployed via a helm release. <br/> Note: If identity is disabled it is likely that the Helm chart installation fails, if you not disable the Identity integration. Make sure to set global.identity.auth.enabled=false AND optimize.enabled=false, since Optimize is highly depending on Identity. | `true` |

Reviewer: ```suggestion
| | `enabled` |  If true, the Identity deployment and its related resources are deployed via a helm release. <br/> Note: Identity is required by Optimize. If Identity is disabled, then Optimize will be unusable. If you don't need Optimize, then make sure to disable both: set global.identity.auth.enabled=false AND optimize.enabled=false. | `true` |
```
